x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/IndexDeprecationChecks.java

```diff
@@ -119,12 +119,12 @@ static DeprecationIssue frozenIndexSettingCheck(IndexMetadata indexMetadata, Clu
         if (Boolean.TRUE.equals(isIndexFrozen)) {
             String indexName = indexMetadata.getIndex().getName();
             return new DeprecationIssue(
-                DeprecationIssue.Level.WARNING,
-                "index ["
-                    + indexName
-                    + "] is a frozen index. The frozen indices feature is deprecated and will be removed in a future version",
+                DeprecationIssue.Level.CRITICAL,
+                "Index [" + indexName + "] is a frozen index. The frozen indices feature will be removed in version 9.0.",
```

Comment:
I don't think we need to remove the "is deprecated and" part of this, since it's still deprecated. We don't want to give the impression we're removing something abruptly.